docs/configuration.md

```diff
@@ -111,7 +111,7 @@ Any flag that has a value that is a comma separated list on the command line bec
 
 #### Gossip Seed
 
-* Description: When using the gossip [Cluster Type]({{< ref "#cluster-type" >}}), this specifies which internal host should be used to initialize membership in the cluster. Typcially this can be the address of any available host in the cluster. For example, when starting a three-node cluster made up of `node0`, `node1`, and `node2`, the `gossip-seed` for all three nodes can be configured to be the address of `node0`.
+* Description: When using the gossip [Cluster Type]({{< ref "#cluster-type" >}}), this specifies which internal host(s) should be used to initialize membership in the cluster. Typcially this can be the address of any available host in the cluster. You may enter multiple seeds by separating them with a comma. For example, when starting a three-node cluster made up of `node0`, `node1`, and `node2`, the `gossip-seed` for all three nodes can be configured to be the address of `node0`.
```

Comment:
I think we should remove the "When using...," clause, and just start with "This specifies..."

Reply:
Also, what do you think about changing changing `Config.Gossip.Seed` to `Config.Gossip.Seeds`? We may want to support both for a period of time, but change the docs to `Seeds`. Then we could also define `Seeds` as `[]string` the way we did with `Hosts`, and the toml could be a true list instead of a string with commas.

Reply:
That's a good idea; I'll do that.